tenable/ad/widget/api.py

```diff
@@ -0,0 +1,260 @@
+'''
+Widgets
+======
+
+Methods described in this section relate to the the about API.
+These methods can be accessed at ``TenableAD.widget``.
+
+.. rst-class:: hide-signature
+.. autoclass:: WidgetsAPI
+
+    .. automethod:: create
+    .. automethod:: delete
+    .. automethod:: details
+    .. automethod:: update
+    .. automethod:: list
+    .. automethod:: widget_options_details
+    .. automethod:: define_widget_options
+'''
+
+from typing import List, Dict
+
+from tenable.ad.widget.schema import WidgetSchema, WidgetOptionSchema
+from tenable.base.endpoint import APIEndpoint
+
+
+class WidgetsAPI(APIEndpoint):
+    _path = 'dashboards'
+    _schema = WidgetSchema()
+
+    def list(self, dashboard_id: str) -> List[Dict]:
+        '''
+        Retrieve all widgets
+
+        Returns:
+            list:
+                The list of widget objects
+
+        Examples:
+
+            >>> tad.widgets.list()
+        '''
+        return self._get(f"{dashboard_id}/widgets")
+
+    def create(self,
+               dashboard_id: str,
+               pos_x: int,
+               pos_y: int,
+               width: int,
+               height: int,
+               title: str) -> List[Dict]:
+        '''
+        Create a new widget
+
+        Args:
+            dashboard_id:
+                The dashboard instance identifier.
+            pos_x:
+                x-axis position for widget.
+            pos_y:
+                y-axis position for widget.
+            width:
+                width of widget.
+            height:
+                height of widget.
+            title:
+                title for widget.
+
```

Comment:
1 return, not 2

Reply:
This has been resolved in latest commit.